ducphanduy/style.css

```diff
@@ -1,47 +1,47 @@
 body {
-	background-color: lightblue;
+    background-color: lightblue;
 }
 
 #button {
-	text-decoration: none;
-	border: solid 1px black;
-	padding: 8px;
-	color: white;
-	border-radius: 3px;
+    text-decoration: none;
+    border: solid 1px black;
+    padding: 8px;
+    color: white;
+    border-radius: 3px;
 }
 
 #query {
-	width: 30em;
-	height: 30px;
-	font-size: 18px;
-	border-radius: 5px;
+    width: 30em;
+    height: 30px;
+    font-size: 18px;
+    border-radius: 5px;
 }
 
 #button:hover {
-	color: lightblue;
-	background-color: white;
+    color: lightblue;
+    background-color: white;
 }
 
 table {
-  	color: black;
-  	font-size: 20px;
-  	margin-top: 20px;
+    color: black;
+    font-size: 20px;
+    margin-top: 20px;
 }
 
 td {
-  	border: 1px solid white;
-  	padding: 3px;
-  	border-radius: 2px; 	
+    border: 1px solid white;
+    padding: 3px;
+    border-radius: 2px;     
 }
 
 tr {
-	width: 40px;
+    width: 40px;
 }
 
 td a {
-	text-decoration: none;
+    text-decoration: none;
 }
 
 tr:hover {
-  	background-color: #6666ff;
+    background-color: #6666ff;
 }
```

Comment:
yes I am adding this bear and some other bears as well and will have a separate PR (since its going to take some time) . and I thought it was more important to get build fixed.